src/main/java/com/metsci/laproc/display/BasicWindow.java

```diff
@@ -0,0 +1,86 @@
+package com.metsci.laproc.display;
+
+import com.jogamp.opengl.util.FPSAnimator;
+import com.metsci.glimpse.canvas.NewtSwingGlimpseCanvas;
+import com.metsci.glimpse.docking.*;
+import com.metsci.laproc.ROCCurvePlot;
+import com.metsci.laproc.plotting.Axis;
+import com.metsci.laproc.plotting.BasicGraph;
+import com.metsci.laproc.plotting.Curve;
+import com.metsci.laproc.plotting.Graph;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Created by porterjc on 9/22/2016.
```

Comment:
To keep what we submit consistent, before merging to master it is probably a good idea to have JavaDocs in our code.